cloud/awsprovider.go

```diff
@@ -437,6 +478,20 @@ func (aws *AWS) DownloadPricingData() error {
 					}
 					aws.ValidPricingKeys[key] = true
 					aws.ValidPricingKeys[spotKey] = true
+				} else if strings.HasPrefix(product.Attributes.UsageType, "EBS:Volume") {
```

Comment:
You probably want contains here, as only us-east-1 is prefixed in this fashion.
For example, here's a us-west2 usagetype:
 "usagetype" : "USW2-EBS:VolumeUsage.gp2",

This may be related to your observation that not all costPerIOOperation is being set, though I am still investigating that.

Reply:
Addressed this by dynamically adding the correct region prefix to UsageTypes in regions which are not us-east-1. Included a map which maps region identifiers (us-east-2) into these UsageType region codes (USE2).